### PR TITLE
NEWS for v238

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -16,6 +16,127 @@ CHANGES WITH 238 in spe:
...
+          which can be used to migrate foreign processes to scope and service
+          units. The primary user for this new API is systemd itself: the
+          systemd --user instance uses this call of the systemd --system
+          instance to migrate processes if it itself gets the request to
+          migrate processes and the kernel refuses this due to access
+          restrictions.  Thanks to this "systemd-run --scope --user …" works
+          again in pure cgroups v2 environments when invoked from the user
+          session scope.
+
+        * A new TemporaryFilesystem setting can be used to mask out part of the
```
Comment (yuwata) [inline NEWS]: Typo: `TemporaryFileSystem=`

File: NEWS [hunk 2]
```diff
@@ -16,6 +16,127 @@ CHANGES WITH 238 in spe:
...
+        * Non-service units are now started with KeyringMode=shared by default.
+          This means that mount and swapon and other mount tools have access
+          to keys in the main keyring.
+
+        * /sys/fs/bpf is now mounted automatically.
+
+        * QNX virtualization is now detected by systemd-detect-virt and may
+          be used in ConditionVirtualization=.
+
+        * IPAccounting may now be enabled also for slice units.
```
Comment (yuwata) [inline NEWS]: `IPAccounting=`, missing `=`.

**Comment by keszybz** [comment]
I force-pushed with fixes for the typos to force semaphore to run again. I updated the configuration, so it should pass now.

**Comment by keszybz** [comment]
OK, let's merge.

**Pull request merged**